### PR TITLE
Ctar1 fixes

### PR DESCRIPTION
**Pull request by maxnoe**
Changes to make it fully work with EVBv6 data, not yet ready, will update as more test data is taken

Missing:

- [X] Read info on event-preprocessing done by EVB
- [ ] Apply only needed steps
- [ ] Missing module handling in code branch for new data format data (do we need this?)

### CODE REVIEW

**Comment by maxnoe** [comment]
This PR is already pretty large and is enough to get analysis working of data taken with EVBv6 without corrections applied or with manually specifying which corrections have already been applied.

Could you please review @FrancaCassol @moralejo or @SeiyaNozaki ?

I'll open another PR for the missing module and the automatic detection of calibration steps already performed by EVB

File: src/ctapipe_io_lst/__init__.py [hunk 1]
```diff
@@ -437,40 +467,76 @@ def create_subarray(tel_id=1, reference_location=None):
 
     def fill_from_cta_r1(self, array_event, zfits_event):
         tel_id = self.tel_id
+        scale = self.data_stream.waveform_scale
+        offset = self.data_stream.waveform_offset
+        pixel_id_map = self.camera_config.pixel_id_map
+
+        # FIXME: missing modules / pixels
```
Comment (moralejo) [inline src/ctapipe_io_lst/__init__.py:474]: So this will not work yet with  data with missing modules?

**Review by moralejo — APPROVED** [review]
Had a look at seems ok to me, but better if sb else checks it too...

File: src/ctapipe_io_lst/__init__.py (hunk 1 above)
Comment (maxnoe) [reply src/ctapipe_io_lst/__init__.py:474]: No, not yet, see my comment above. 

I'd like to separate that part into a second PR, with proper dummy test data and so on. This one will get pretty large if I want to include everything.

**Pull request merged**